### ISSUE TITLE
test not found

### ISSUE THREAD

**Issue by vitaly-zdanevich** [heroku/force#414]
```
$ force test MyTest
ERROR: Test classes specified not found: [MyTest]
```

Also I tried with namespace with the same result.

**Comment by KThompso** [comment]
Are you certain that you pushed the source file into your Salesforce org?  The `force test` command doesn't push anything to the org, it only runs tests that exist in your organization already.  Can you post the output of:

`$ force describe -t=metadata -n=ApexClass|grep -i MyTest`

**Comment by qrcodes** [comment]
Can confirm that it doesn't work. Tests won't run regardless of whether or not they're namespaced.  It's not possible to run any test.

With namespace:
`ERROR: This class name's value is invalid:  Namespace.Namespace. Provide the name of an Apex class that has test methods.`

Without namespace:
`ERROR: Test classes specified not found: [testclassname]`

Since this functionality is broken, I had to resort to using an older version of ForceCLI.

**Comment by cwarden** [comment]
@qrcodes, does `force test -namespace Namespace testclassname` work?

**Comment by qrcodes** [comment]
@cwarden Yes, that worked. Didn't realize the parameters had changed. Thanks.
